Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dev-dependencies]
 near-sdk = "3.1.0"
-near-sdk-sim = "3.1.0"
+near-sdk-sim = "3.1.1"
```

Comment:
Small nit but do you mind switching to this version in the fungible token contract too? Just in case someone copies just that part of the example

Reply:
hi @austinabell , thanks for the review

I have updated the version and committed `Cargo.lock`. Let me know if I missed something.

Reply:
Also updated `./ft/Cargo.toml` ✅